cmd/gb-vendor/vendor/manifest.go

```diff
@@ -28,6 +29,18 @@ func (m *Manifest) AddDependency(dep Dependency) error {
 	return nil
 }
 
+// RemoveDependency removes a Dependency from the current Manifest
```

Comment:
full stop at the end of these lines (I know I'm not that consistent with this, but I should get better).